### PR TITLE
Feature/new animations

### PR DESCRIPTION
**Pull request by Lena0020**
Added animations for all enemies as well as for player close-range attacks.
Also added Energy Beam Sprite.
Here's what the enemies look like:
image
We have all the animations added from this pack:
https://penusbmic.itch.io/sci-fi-character-pack-12
dying/attacking in different ways/appearing/idle/taking damage/moving
Jira US:
https://s24109.atlassian.net/browse/SCRUM-51

### CODE REVIEW

**Comment by FrozzLab** [comment]
Everything seems good, waiting for second review now

**Review by FrozzLab — APPROVED** [review]

**Comment by WildCup** [comment]
can you please add some screenshots to description? and if you could please summarize what animations we have and what we still need

**Comment by WildCup** [comment]
please add a link to a task and in the future name a branch feature/{you-us-id}-description as mention in wiki. Otherwise we don't know what this PR should cover. (I mean okay in this case we can all guess because of small size of the project but it's just a good practice :) )

**Pull request merged**
